tests/PersonBearbeiten.spec.ts

```diff
@@ -195,4 +195,188 @@ test.describe(`Testfälle für die Administration von Personen": Umgebung: ${pro
             await personDetailsView.checkLockDateTo(sperrDatumBis); // Der Benutzer muss gesperrt sein bis zun dem eingegebenen Datum
         })
     })
+
+    test("Gesamtübersicht für einen Benutzer als Schueler öffnen und Unsichtbarkeit des 2FA Abschnitts prüfen", {tag: [LONG, STAGE]}, async ({ page }) => {
```

Comment:
Ich würde vorschlagen, neue Tests **immer** nur in long zu packen damit wir erstmal prüfen können, ob der neue Test stabil ist. Ansonsten behindert ein flaky Test das stage Release.